File: Assignment_3/Assignment_3.c




#include<stdio.h>


void PrintEven(int iNo)
{
    int iCnt = 0;
    
   if(iNo <= 0 )
    {
        iNo = -iNo;
    }
    else
    {
        for(iCnt = 1 ; iCnt < iNo; iCnt++)
        {
            if(iNo % iCnt == 0 && iCnt % 2 == 0 )
            {
                printf("  %d  ",iCnt);
            
            }
            
        }
        printf("\n");
    }
   
}

int main()
{
    int iValue = 0;
    printf("Enter Number :  ");
    scanf("%d", &iValue);
    
    PrintEven(iValue);

    return 0;

}

File: Assignment_11/Assignment_java/Assignment_3.java
import java.util.*;

class Pattern 
{
    public void Display(int iNo)
    {
        int iCnt = 0;
       
        for(iCnt = 1 ; iCnt <= iNo ; iCnt++)
        {
            System.out.print(iCnt+" * ");
        }
        System.out.println();
    }
}
class Assignment_3
{


    public static void main(String A[])
    {   
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number : ");
        int iValue = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue);
    }
}